sdlib/d/gc/thread.d

```diff
@@ -126,19 +127,18 @@ private:
 	Mutex mThreadList;
 	ThreadRing registeredThreads;
 
-	Mutex stopTheWorldMutex;
+	shared SharedLock createThreadLock;
```

Comment:
It doesn't matter that this is used when thread are created. It's purpose is to gate the stop the world feature. And it happens that you don't want to stop the world mid thread creation.

Reply:
renamed to stopTheWorldLock.